types/zoomist/index.d.ts

```diff
@@ -0,0 +1,181 @@
+// Type definitions for zoomist 1.1
+// Project: https://github.com/cotton123236/zoomist
+// Definitions by: scriptSQD <https://github.com/scriptSQD>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Minimum TypeScript Version: 4.1
+
+export = Zoomist;
```

Comment:
```suggestion
export as namespace Zoomist;

export = Zoomist;
```
https://github.com/cotton123236/zoomist/tree/45da887fa2474c0fa2e1635cf1796250290287e5#script-tag-include
I've missed this one, it looks like proper UMD module:
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library

Reply:
Oh I was really struggling with that stuff =) Let me check it on local project and commit if eveything's fine!

Reply:
@peterblazejewicz Thanks again! Commited changes and now waiting for library's author